docs/src/userguide/numpy_tutorial.rst

```diff
@@ -125,10 +124,6 @@ like::
 
     $ gcc -shared -pthread -fPIC -fwrapv -O2 -Wall -fno-strict-aliasing -I/usr/include/python2.7 -o yourmod.so yourmod.c
 
-``gcc`` should have access to the NumPy C header files so if they are not
-installed at :file:`/usr/include/numpy` or similar you may need to pass another
-option for those.
```

Comment:
Removing this paragraph removes information that users would need for manual compilation: "call `numpy.get_include()` and add the result to your C include paths."

Reply:
Does the user have to do this for memoryviews too? I just now that it's the case when using the old numpy buffer syntax.

Reply:
Ah, true. The include path is only required when you do `cimport numpy`. (Meaning, it's also independent of the buffer syntax and only depends on the `numpy` cimport (plain Python imports are also fine).

Reply:
I added the note about the numpy headers and added another line just after that it's needed only if doing `cimport numpy`.